_posts/2019-11-27-cockpit-208.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 208
+author: pitti
+date: 2019-11-27
+tags: cockpit linux
+slug: cockpit-208
+category: release
+summary: Cockpit with Machines, file system creation, and crash reporting improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 208.
+
+### Storage: Drop "default mount point" concept
+
+Cockpit no longer offers the confusing "Default" choice when mounting a filesystem. Now every new filesystem requires the specification of a mount point. Additionally, Cockpit hides the distinction between configuration (`/etc/fstab`) and run-time state (`/proc/mounts`). Changes made in Cockpit always apply to both the configuration and run-time state. When configuration and run-time state differ from each other, Cockpit will warn about that and will let you bring them back in sync.
+
+![Storage format dialog](/images/storage-format-mountpoint.png)
+
+### Machines: Support transient virtual networks and storage pools
+
+Transient resources are resources which exist only while they are running and cease to exists once they stop running/are deactivated. Cockpit now disables or hides functionality like autostart or deletion for transient resources, as these would fail:
+
+![Transient virtual network](/images/machines-transient-network.png)
+
+### Machines: A size sliders for VM creation
+
+The "Create New Virtual Machine" dialog now has sliders to configure the memory and disk sizes:
+
+![Machine creation size sliders](/images/machines-size-sliders.png)
+
+### Logs: Use reportd for reporting crashes
+
+Cockpit previously sent problem/crash reports directly to [ABRT Analytics](https://github.com/abrt/faf) when reporting them on the Logs page. That got changed to using the new "reportd" framework. This supports reporting to more places, such as [Bugzilla](https://bugzilla.redhat.com/), and handles authentication in a better way.
```

Comment:
"got changed" is probably too informal for release notes.  Maybe something like "Cockpit now uses the reportd framework" or so?

Reply:
Thanks, fixed.